integrations/timm/README.md

```diff
@@ -0,0 +1,123 @@
+<!--
+Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# SparseML-rwightman/pytorch-image-models integration
+This directory provides a SparseML integrated training script for the popular
+[rwightman/pytorch-image-models](https://github.com/rwightman/pytorch-image-models)
+repository also known as [timm](https://pypi.org/project/timm/).
+
+Using this integration, you will be able to apply SparseML optimizations
+to the powerful training flows of the pytorch-image-models repository.
+
+Some of the tasks you can perform using this integration include, but are not limited to:
+* model pruning
+* quantization-aware-training
+* sparse quantization-aware-training
+* sparse transfer learning
+
+## Installation
+Both requirements can be installed via `pip` or can be cloned
+and installed from their respective source repositories.
+
+```bash
+pip install git+https://github.com/rwightman/pytorch-image-models.git
+pip install sparseml[torchvision]
+```
+
+
+## Script
+`integrations/timm/main.py` modifies
```

Comment:
one more small change, can we get the main file renamed to train.py? That way we have a one to one mapping for the file(s) from the integration with sparseml